packages/cpp/src/browser/cpp-language-client-contribution.ts

```diff
@@ -27,6 +27,13 @@ import { CPP_LANGUAGE_ID, CPP_LANGUAGE_NAME, HEADER_AND_SOURCE_FILE_EXTENSIONS }
 import { CppBuildConfigurationManager, CppBuildConfiguration } from "./cpp-build-configurations";
 import { CppBuildConfigurationsStatusBarElement } from "./cpp-build-configurations-statusbar-element";
 
+// Clangd extension to set clangd-specific "initializationOptions" in the
```

Comment:
I think we use jsdoc comments (`/** ... */`) to document types.

Reply:
done